rustfmt.toml

```diff
@@ -0,0 +1,14 @@
+# See https://github.com/rust-lang/rustfmt for more information
+
+# `Foobar { foo, bar }` is more readable than `Foo { foo: foo, bar: bar }`
+use_field_init_shorthand = true
+
+# Forces let else blocks to always be their own line(s)
+single_line_let_else_max_width = 0
```

Comment:
This is a new introduction. It's quite minor and so I don't personally care much either way.

Maybe someone else has opinions as this is new.